test/routes/save-record.route.test.js

```diff
@@ -175,6 +222,8 @@ const _createMocks = () => {
     .mockReturnValue('THE_SESSION_COOKIE')
 
   ODataService.updateRecord = jest.fn()
+
+  PaymentService.lookupPayment = jest.fn()
```

Comment:
I don't think that you need this because the ODataService.updateRecord is created an every test anyway.

Reply:
Fixed